tests/management/group/test_view.py

```diff
@@ -1003,6 +999,35 @@ def test_delete_custom_default_group(self):
         response = client.delete(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
 
+        actual_call_arg = mock_method.call_args[0][0]
+
+        to_add = actual_call_arg["relations_to_add"]
+
+        tenant_mapping = TenantMapping.objects.get(tenant=self.tenant)
+
+        def assert_group_tuples(tuple_to_replicate):
+            relation_tuple = relation_api_tuple(
```

Comment:
I wonder if we could reuse some of the existing concepts more in tests. I find these tests a little hard to follow and update personally but it might just be me. For example, instead of new methods to create tuples like `relation_api_tuple`, can we use the `Relationship` type already used, or `create_relationship` method, or use `InMemoryTuples` and the `RelationTuple` type?

Maybe we have too many ways of doing the same thing... 🤔. It makes running into new tests sorta like starting over to learn how "those" tests do it.

Reply:
I agree with there is place for improvements and unification. 

btw I looks there is better to [way](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args) for call_args.

Let's consider mentioned points and I will create ticket for it 👍